tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist=py36,py37,py38
+envlist=py37,py38,py39,py310,py311,py312
```

I believe Python 3.7 is EOL.

```suggestion
envlist=py38,py39,py310,py311,py312
```